remove GeoshapeHelper from JanusGraphGremlinPlugin

After the latest merge to master:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project janusgraph-all: Compilation failure
[ERROR] /Users/pluradj/src/github/JanusGraph/janusgraph/pulls/janusgraph-all/src/main/java/org/janusgraph/graphdb/tinkerpop/plugin/JanusGraphGremlinPlugin.java:[54,37] cannot find symbol
[ERROR] symbol:   class GeoshapeHelper
[ERROR] location: package org.janusgraph.core.attribute
[ERROR] -> [Help 1]
```

Travis was passing, so seems like this code path in `janusgraph-all` isn't covered.

--- a/janusgraph-all/src/main/java/org/janusgraph/graphdb/tinkerpop/plugin/JanusGraphGremlinPlugin.java
+++ b/janusgraph-all/src/main/java/org/janusgraph/graphdb/tinkerpop/plugin/JanusGraphGremlinPlugin.java
@@ -51,7 +51,6 @@ import org.janusgraph.core.attribute.Cmp;
 import org.janusgraph.core.attribute.Contain;
 import org.janusgraph.core.attribute.Geo;
 import org.janusgraph.core.attribute.Geoshape;
-import org.janusgraph.core.attribute.GeoshapeHelper;
 import org.janusgraph.core.attribute.JtsGeoshapeHelper;
 import org.janusgraph.core.attribute.Text;
 import org.janusgraph.core.schema.ConsistencyModifier;
@@ -154,7 +153,6 @@ public class JanusGraphGremlinPlugin extends AbstractGremlinPlugin {
         CLASS_IMPORTS.add(Contain.class);
         CLASS_IMPORTS.add(Geo.class);
         CLASS_IMPORTS.add(Geoshape.class);
-        CLASS_IMPORTS.add(GeoshapeHelper.class);
         CLASS_IMPORTS.add(JtsGeoshapeHelper.class);
         CLASS_IMPORTS.add(Text.class);
 

